uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_annual-entitlement-change-log.scss

```diff
@@ -1,9 +1,9 @@
 .annual-entitlement-change-log-component {
-  table thead td {
+  thead td {
     min-width: 100px;
   }
 
-  table thead td.annual-entitlement-change-log-component__comments {
+  thead .annual-entitlement-change-log-component__comments {
```

Comment:
i thought that the linter didn't like generic tag selectors?

Reply:
@AkA84 it will not like `td.some-class` but just `td` or `thead` in this case is totally fine.